### PR TITLE
Three new token metadata+assets [LENFI, AXO, SNEK]

### PR DESCRIPTION
**Pull request by rebelC0der**
# Hello MELD Team! &#128736;
I am submitting this pull request to add details for the following tokens: Lenfi (LENFI), Axo (AXO), and Snek (SNEK) in the repository. I have followed the prescribed guidelines and ensured that all necessary details are accurate and align with the required data structure. Below is an overview of each token and the associated changes.

- [x] I have checked that the format of my PR follows the contribution guidelines

---

### Lenfi (LENFI)
- Token Name: Lenfi
- Token Ticker: LENFI
- Blockchain: Cardano
- Token Type: Native Token
- Primary Use Case: DeFi Lending Platform

### Organization Details &#127970;
- Organization Name: Lenfi
- Website: [https://lenfi.io](https://lenfi.io)
- Mission & Vision: Provide decentralized lending services on the Cardano blockchain with security and user-centric features.

---

### Axo (AXO)
- Token Name: Axo
- Token Ticker: AXO
- Blockchain: Cardano
- Token Type: Native Token
- Primary Use Case: Decentralized Exchange Platform

### Organization Details &#127970;
- Organization Name: Axo
- Website: [https://www.axo.trade](https://www.axo.trade)
- Mission & Vision: Revolutionize decentralized trading with high efficiency and security on Cardano.

---

### Snek (SNEK)
- Token Name: Snek
- Token Ticker: SNEK
- Blockchain: Cardano
- Token Type: Native Token
- Primary Use Case: Meme Token with community-driven initiatives.

### Organization Details &#127970;
- Organization Name: Snek
- Website: [https://www.snek.com/](https://www.snek.com/)
- Mission & Vision: Build a strong community around a fun and engaging meme token on the Cardano network.

---

### For Additions:
- Community Demand:
  - **Lenfi**: Strong demand within the Cardano DeFi community.
  - **Axo**: Anticipated demand due to its unique decentralized trading approach.
  - **Snek**: Significant grassroots support within the Cardano community.
  
- Unique Value Proposition:
  - **Lenfi**: Decentralized lending with Cardano-native security features.
  - **Axo**: High-efficiency DEX with novel trading features.
  - **Snek**: Community-driven with potential for broad appeal as a meme token.
  
- Technical Robustness:
  - **Lenfi**: Well-audited contracts and active development.
  - **Axo**: Built on a solid Cardano smart contract foundation.
  - **Snek**: Backed by a passionate and technically sound community.

- Compliance and Security:
  - **Lenfi**: Adheres to Cardano's security best practices.
  - **Axo**: Compliant with all Cardano security standards.
  - **Snek**: Follows community-driven security initiatives.

---

## Rationale for Inclusion &#128200;
- Benefits to the Repository:
  - **Lenfi**: Adds a secure DeFi option to the Cardano ecosystem.
  - **Axo**: Introduces an advanced trading platform.
  - **Snek**: Engages the community with a popular meme token.

- Impact on Community:
  - **Lenfi**: Enables secure and decentralized lending.
  - **Axo**: Offers a new trading experience on Cardano.
  - **Snek**: Strengthens community engagement with a fun token.

---

Thank you for considering this pull request. I am confident that these additions will provide significant value to the repository and the wider Cardano community.

### CODE REVIEW

File: networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "axo",
+    "decimals": 6,
```
Comment (dongntp) [inline networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json:3]: This token has 9 decimals, not 6: https://cexplorer.io/asset/asset16gtljf3uzghxyml9cjh9uvwg5jmcz25gy6kelu

Comment (rebelC0der) [reply networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json]: Corrected.

File: networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "axo",
+    "decimals": 9,
+    "mldId": "cardano-AXO-420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f",
+    "orgName": "Axo",
+    "orgUrl": "https://www.axo.trade",
+    "tokenAddress": "420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f",
+    "tokenColorTheme": 21,
+    "tokenIcon": "/static/tokens/AXO.svg",
+    "tokenName": "Axo",
```
Comment (dongntp) [inline networks/cardano/tokens/420000029ad9527271b1b1e3c27ee065c18df70a4a4cfc3093a41a4441584f.json:10]:
```suggestion
    "tokenName": "AXO",
```

File: networks/cardano/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "CMCslug": "snek",
+    "decimals": 6,
```
Comment (maian99) [inline networks/cardano/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b.json:3]: Snek has decimals 0: https://cardanoscan.io/token/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b

Comment (rebelC0der) [reply networks/cardano/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b.json]: Confirm this with a few sources:
1. https://cardanoassets.com/asset108xu02ckwrfc8qs9d97mgyh4kn8gdu9w8f5sxk
2. https://minswap.org/tokens/snek
3. https://adastat.net/tokens/279c909f348e533da5808898f87f9a14bb2c3dfbbacccd631d927a3f534e454b

**Review by dongntp — APPROVED** [review]

**Review by maian99 — APPROVED** [review]

**Pull request merged**